Formula/z/zstd.rb

```diff
@@ -10,7 +10,7 @@ class Zstd < Formula
 
   livecheck do
```

Comment:
```suggestion
  # The upstream repository contains old, one-off tags (5.5.5, 6.6.6) that are
  # higher than current versions, so we check the "latest" release instead.
  livecheck do
```

The `stable` URL is a GitHub tag archive, so we should add a comment explaining why `GithubLatest` is necessary.